nerdlandbot/commands/random.py

```diff
@@ -0,0 +1,132 @@
+
+if not os.path.exists(PONCHO_DIR_NAME):
+    os.makedirs(PONCHO_DIR_NAME)
+class Random(commands.Cog, name="Random"):
+    def __init__(self, bot):
+        self.bot = bot
+
+
+    @commands.command(name="random_user", aliases=["randomuser"], brief="random_user_brief", usage="random_user_usage",
+                      help="random_user_help")
+    async def select_random_user(self, ctx: commands.Context, *, channel_name: str = None):
+        """
+        Selects a random user from the server, channel, or online members.
+        :param ctx: The current context. (discord.ext.commands.Context)
+        :param channel_name: The channel to pick from, 'online' in order to pick an online member from the server. (str)
+        """
+
+        # Error for missing parameter
+        if channel_name is None:
+            msg = translate("random_user_no_channel", await culture(ctx))
+            return await ctx.send(msg)
+
+        # Choose from online members if requested
+        if channel_name == "online":
+            member = pick_random_online_member(ctx)
+            msg = translate("random_user_chosen", await culture(ctx)).format(member.id)
+            return await ctx.send(msg)
+
+        # Sanitize channel name
+        channel = get_channel(ctx,channel_name)
+
+        # Error if channel does not exist
+        if channel is None:
+            msg = translate("membercount_channel_nonexistant", await culture(ctx))
+            return await ctx.send(msg)
+
+        # Error if channel empty
+        if len(channel.members) < 1:
+            msg = translate("membercount_empty_channel", await culture(ctx)).format(channel.id)
+            return await ctx.send(msg)
+
+        # Pick a random user from channel, and report back to user
+        member = random.choice(channel.members)
+        msg = translate("random_user_chosen", await culture(ctx)).format(member.id)
+        await ctx.send(msg)
+
+    @commands.command(name="wombat_pic", aliases = ["wombat","wombatpic"], hidden=True, help="wombat_pic_help")
+    async def cmd_wombat_pic(self, ctx):
+        if not os.path.exists(WOMBATS_DIR_NAME):
+            os.makedirs(WOMBATS_DIR_NAME)
+            current_directory = os.getcwd()
+            warn(f"Directory {os.path.join(current_directory,WOMBATS_DIR_NAME)} is created, put some wombat pictures in it!" )
+            msg = translate("empty_wombat_list", await culture(ctx))
+            return await ctx.send(msg)
+
+        wombat_list = [os.path.join(WOMBATS_DIR_NAME, w) for w in os.listdir(WOMBATS_DIR_NAME)]
+
+        if not wombat_list:
+            msg = translate("empty_wombat_list", await culture(ctx))
+            return await ctx.send(msg)
+
+        await ctx.send(file=discord.File(random.choice(wombat_list)))
+
+    @commands.command(name="poncho", aliases=["poncho_pic","ponchopic"],hidden=True, help="poncho_pic_help")
+    async def cmd_poncho_pic(self, ctx):
+        if not os.path.exists(PONCHO_DIR_NAME):
```

Comment:
Isn't this a duplicate of line 32 ? why make the directory inside the command itself?

Best approach is probably to put it in an init function of the Class itself like here: https://github.com/nerdland-unofficial-fans/nerdlandbot/blob/1ca2ec15b0e3b35db92d984a2d043a4825890d84/nerdlandbot/commands/space_launches.py#L21